packages/cell-toolbar-extension/schema/plugin.json

```
@@ -0,0 +1,177 @@
+      "items": {
+        "type": "string",
+        "pattern": "^[\\w-]+$"
+      }
+    },
+    "floatPosition": {
+      "title": "Floating cell toolbar position",
+      "description": "Place the cell toolbar above the cell. If set, `leftSpace` is ignored.",
+      "oneOf": [
+        {
+          "type": "null"
+        },
+        {
+          "$ref": "#/definitions/position"
+        }
+      ],
+      "default": null
+    },
+    "helperButtons": {
+      "title": "Helper buttons",
+      "description": "Set the list of visible helper buttons.",
+      "default": [
+      ],
+      "oneOf": [
+        {
+          "type": "null"
+        },
+        {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "enum": [
+              "insert-cell-below",
+              "move-cell-down",
+              "move-cell-up",
+              "run-cell-and-select-next"
+            ]
+          },
+          "uniqueItems": true
+        }
+      ]
+    },
+    "leftMenu": {
+      "title": "List of left menu items",
+      "description": "An item is defined by a 'command' name and an 'icon' name + optionally a 'tooltip' and the 'cellType' on which it applies",
+      "oneOf": [
+        {
+          "type": "null"
+        },
+        {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/menuItem"
+          }
+        }
+      ],
+      "default": [
+        {
+          "command": "notebook:duplicate-below",
+          "icon": "@jupyterlab/cell-toolbar:duplicate",
+          "tooltip": "Duplicate Cell"
+        },
+        {
+          "command": "notebook:move-cell-up",
+          "icon": "@jupyterlab/cell-toolbar:move-up",
+          "tooltip": "Move Cell Up"
+        },
+        {
+          "command": "notebook:move-cell-down",
+          "icon": "@jupyterlab/cell-toolbar:move-down",
+          "tooltip": "Move Cell Down"
+        },
+        {
+          "command": "notebook:insert-cell-above",
+          "icon": "@jupyterlab/cell-toolbar:add-above",
+          "tooltip": "Insert Cell Above"
+        },
+        {
+          "command": "notebook:insert-cell-below",
+          "icon": "@jupyterlab/cell-toolbar:add-below",
+          "tooltip": "Insert Cell Below"
+        },
+        {
+          "command": "notebook:delete-cell",
+          "icon": "@jupyterlab/cell-toolbar:delete",
+          "tooltip": "Delete Cell"
+        }
+      ]
+    },
+    "leftSpace": {
+      "title": "Cell toolbar empty left space size",
+      "description": "Size of the empty space left of the cell toolbar in px. Ignored if `floatPosition` is not null.",
+      "type": "number",
+      "minimum": 0,
+      "default": 0
+    },
+    "rightMenu": {
+      "title": "List of left menu items",
+      "description": "An item is defined by a 'command' name and an 'icon' name + optionally a 'tooltip' and the 'cellType' on which it applies",
+      "oneOf": [
+        {
+          "type": "null"
```

Do we really need the `null` type here? Would not check for an empty list suffice?

```suggestion
          "type": "null",
          "title": "Right menu disabled"
```